theme/js/dynamic-scripts.js

```diff
@@ -8,19 +8,41 @@ $(document).ready(function() {
     getDonationSumData();
 });
 
+$.fn.extend({
+    toggleText: function(a, b){
+        return this.text(this.text() == b ? a : b);
+    }
+});
+
 function getBlogCardComponentData() {
     $.getJSON('theme/json/posts.json', function(data) {
         // Run function for each card. The post variable here represents the json dataset.
         $.each(data, function(i, post) {
+            if ( i >= 3) {
+                var visibility = 'hidden';
+            } else {
+                var visibility = 'visible';
+            }
             $('.card-component-container .row-container--horizontal')
             // Insert the following markup and json data into the card-component-container div.
-                .append('<div class="card-wrapper"><div class="card card-one clickable card--horizontal"><div class="card-image-wrapper"><img src="'+post.imageLink+'" alt="'+post.imageAltText+'" class="card-image responsive-image"></div><div class="card-text-container"><h3>'+post.title+'</h3><p>'+post.datePosted+'</p><p>'+post.summary+'</p><a href="'+post.link+'?postId='+post.id+'" aria-label="'+post.title+'">'+post.linkText+'</a></div></div></div>');
+                .append('<div class="card-wrapper '+visibility+'"><div class="card clickable card--horizontal"><div class="card-image-wrapper"><img src="'+post.imageLink+'" alt="'+post.imageAltText+'" class="card-image responsive-image"></div><div class="card-text-container"><h3>'+post.title+'</h3><p>'+post.datePosted+'</p><p>'+post.summary+'</p><a href="'+post.link+'?postId='+post.id+'" aria-label="'+post.title+'">'+post.linkText+'</a></div></div></div>');
         });
     });
+
     setTimeout(function(){
         clickableElement($('.card-component-container--horizontal .card-wrapper'));
+        showMoreButtonFunctionality();
```

Comment:
A better name for this function might be **showMoreContent();** because that's what the function does.

Reply:
done